app/commands/v2/represent_downstream.rb

```diff
@@ -48,7 +48,7 @@ def downstream_live(content_id, locale)
             content_id: content_id,
             locale: locale,
             payload_version: event.id,
-            message_queue_update_type: "links",
+            message_queue_event_type: "links",
```

Comment:
maybe we should rename this - I think "links" was only used as a hack as we didn't have an alternative option whereas it's a type of republish